docs/docs/components/calendar-input.md

```diff
@@ -115,6 +115,20 @@ Ethiopic calendar (with narrow day names, short day names and localised to Engli
     </style>
 </Demo>
 
+The wrapper is a custom component that wraps the calendar and input field. It is not part of the library. This component is called `CalendarDemo` in our code, and can be found in the [source code](https://github.com/dhis2/ui/blob/08c3e684970437048179179d8cfad5e2edcfc814/docs/src/components/CalendarDemo.jsx#L6).
```

Comment:
While I get what this is referring, I also think it would be confusing to users. If we want to show a demo code in the snippet, I think it would be better to just use the <Calendar> component and let users figure out that there is some kind of wrapper around it that is showing some extra output